include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -305,9 +305,10 @@ compared to 1*64 when the hasLeadingOffset is false.
           int perPhase = 128 / (shapePerCTA[order[0]] * 4 / dotOpEnc.getKWidth());
           perPhase = std::max<int>(perPhase, 1);
           std::vector<size_t> matShape = {8, 8, 4 * dotOpEnc.getKWidth()};
-          // for now, disable swizzle when using transposed int8 tensor cores
-          if ((32 / typeWidthInBit != dotOpEnc.getKWidth()) && order[0] == inner)
-            return get(context, 1, 1, 1, order, CTALayout);
+          int vecWidth = 32 / typeWidthInBit;
+          if (vecWidth != dotOpEnc.getKWidth() && order[0] == inner) {
+              perPhase = std::max<int>(perPhase, 2 * vecWidth);
+          }
```

Comment:
I imagine the intent is to cause the TTGIR -> LLVM lowering to change?

If so, can we add a lit test that calls the TTGIR -> LLVM conversion pass and checks that this is doing what we expect it to do?

Reply:
This impacts an earlier part of the compiler pipeline, namely "reduce-data-duplication". I added a lit test for the updated swizzling. Before the change, the assigned values for the shared layout would be vec=1, perPhase=1, and maxPhase=1.